run.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+DB_FILE=osCam/db.sqlite3
+EMAIL_FILE=.env
```

Comment:
If we do create a form for email address and secret key in the web app, then we can get rid of the .env file, this would streamline how users interact with our product.

Reply:
I completely agree with you. I was just trying to remove the step of the user creating files and some possible human error. The best practice would be to just use the email the users signs up with when first accessing the web app.